source/plugins/go/src/telemetry.go

```diff
@@ -302,6 +302,14 @@ func SendContainerLogPluginMetrics(telemetryPushIntervalProperty string) {
 		osType := os.Getenv("OS_TYPE")
 		if osType != "" && strings.EqualFold(osType, "windows") {
 			telegrafEnabled["IsTelegrafEnabled"] = os.Getenv("TELEMETRY_CUSTOM_PROM_MONITOR_PODS") // If TELEMETRY_CUSTOM_PROM_MONITOR_PODS, then telegraf is enabled
```

Comment:
use the variable isTelegrafEnabled = os.Getenv("TELEMETRY_CUSTOM_PROM_MONITOR_PODS") and assign this value telegrafConfig["IsTelegrafEnabled"] = isTelegrafEnabled